packages/compiler/test/basic/lt-gt-text.ts

```diff
@@ -40,7 +40,7 @@ test.before(async () => {
 
 test('< and > as raw text', () => {
   assert.ok(result.code, 'Expected to compile');
-  assert.ok(result.diagnostics.length, 0, 'Expected no diagnostics');
```

Comment:
wacky that uvu doesn't complain about extra args - https://github.com/lukeed/uvu/blob/master/docs/api.assert.md#okactual-t-msg-message